sw/device/silicon_creator/lib/BUILD

```diff
@@ -396,24 +404,30 @@ cc_test(
     ],
 )
 
-opentitan_functest(
+opentitan_test(
     name = "crc32_functest",
     srcs = ["crc32_functest.c"],
+    exec_env = EARLGREY_TEST_ENVS,
+    verilator = verilator_params(
```

Comment:
This timeout was not there before, was it too small or did the default for verilator changed in your PR?

Reply:
I experienced a timeout when I tested this rule.  I _also_ experience a timeout (on my workstation) on this test prior to the rule refactoring.